app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt

```diff
@@ -103,10 +104,20 @@ abstract class BaseContainerFragment : BaseResourceFragment() {
     private fun openIntent(items: RealmMyLibrary, typeClass: Class<*>?) {
         val fileOpenIntent = Intent(activity, typeClass)
         if (items.resourceLocalAddress?.contains("ole/audio") == true || items.resourceLocalAddress?.contains("ole/video") == true) {
+            Log.d("openIntent", "Touched File Before Passing: ${items.resourceLocalAddress}")
```

Comment:
I think log statements should be removed after finished debugging.

Reply:
@rlam20 remove the logs